web/cobrands/bathnes/base.scss

```diff
@@ -64,11 +73,39 @@
         margin: 0;
         font-size: 1.1em;
     }
+
+    #geolocate_link {
+        background-color: $bathnes-white;
+
+        &:hover {
+            background-color: $bathnes-primary;
```

Comment:
Is it okay to have the background change to the same as the bar when hovered? Wondered if it should be slightly different.

Reply:
So as long the contrast pass between the text and the background colour, and there is a noticeable difference between the states, it should be fine. For colour blind users works well due the luminosity difference between the hover and the default state.

In terms of style I agree, I fixed up so now the hover state has a light blue version.

Thanks for the feedback @dracos and I squashed the commit you mentioned before =)